zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -122,45 +100,12 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
 
         cb.range_lookup(depth.expr(), 1024);
 
-        // Lookup values from stack
-        cb.stack_pop(gas_word.expr());
-        cb.stack_pop(code_address_word.expr());
-
-        // `CALL` opcode has an additional stack pop `value`.
-        cb.condition(is_call.expr() + is_callcode.expr(), |cb| {
-            cb.stack_pop(value.expr())
-        });
-
-        cb.condition(not::expr(is_call.expr() + is_callcode.expr()), |cb| {
```

Comment:
this constraint seems missing in `CommonCallGadget`, it ensure value is zero when not call/call_code.

Reply:
fixed in [ec10a89](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1023/commits/ec10a8953a6fdfa91f169e2fdfa9a08dbb2ac0ab)